components/Images/images.css

```diff
@@ -0,0 +1,18 @@
+@import "../../style_guide.css";
+
+body {
+  background-color: var(--bg-color);
+  color: var(--txt-color);
+}
```

Comment:
Avoid adding element styles in the component.

Reply:
yea this is only for the components I am building temporarily. I will change that once I build the doc site

Reply:
ok.

Reply:
ok